app/src/main/java/com/example/trainingintive/viewmodels/SplashViewModel.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.trainingintive.viewmodels
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.trainingintive.navigators.Navigator
+import com.example.trainingintive.navigators.SplashNavigator
+import com.example.trainingintive.util.Event
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
+import io.reactivex.rxjava3.core.Single
+import io.reactivex.rxjava3.disposables.Disposable
+import io.reactivex.rxjava3.schedulers.Schedulers
+import java.util.concurrent.TimeUnit
+
+class SplashViewModel : ViewModel() {
+
+    val navigator: Navigator = SplashNavigator()
+    private val _event = MutableLiveData<Event>()
+    val event: LiveData<Event> = _event
+    val downloadData: Disposable
+
+    init {
+        downloadData = Single.timer(5, TimeUnit.SECONDS)
+            .flatMap {
+                _event.postValue(Event.Downloading)
+                Single.fromCallable({ downloadData() })
+            }
+            .subscribeOn(AndroidSchedulers.mainThread())
+            .observeOn(Schedulers.io())
+            .subscribe(
+                { _event.postValue(Event.DisplayUserScreen) },
+                { _event.postValue(Event.Error) }
+            )
+    }
+
+    fun downloadData() {}
+
+    fun start() {
```

Comment:
This method shouldn't exist. Events should arrive to your `Navigator` without explicit call to `start()` from Activity. Try to achieve call  `eventLiveData.observe(...)` in your navigator.